### PR TITLE
Integration of intensity at per-frame positions

### PR DESCRIPTION
**Pull request by uellue**
For Till :-)

## Contributor Checklist:

* [x] I have added myself to [the creators.json file](https://github.com/LiberTEM/LiberTEM-blobfinder/blob/master/packaging/creators.json)
* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM-blobfinder/tree/master/docs/source/changelog) for my contribution
* [x] I have added/updated documentation for all user-facing changes
* [x] I have added/updated test cases

### CODE REVIEW

**Comment by uellue** [comment]
Example is deferred in #31 since it requires a proper dataset

File: src/libertem_blobfinder/udf/integration.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+class IntegrationUDF(UDF):
+    def __init__(self, centers, pattern):
+        '''
+        Integrate peak intensity at positions that are specified for each frame.
+
+        Parameters
+        ----------
+        centers : AUXBufferWrapper
```
[the remark this answers is not in the document]
Comment (uellue) [reply src/libertem_blobfinder/udf/integration.py:15]: Good point! :-) Done now.

**Comment by uellue** [comment]
Hi @sk1p, could you perhaps have a look if the example is clear enough? :-)

**Comment by sk1p** [comment]
Looks good, thanks!

**Pull request merged**